modules/shared/components/molecules/imageCaption/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+import type { FC, ReactElement } from 'react';
+import type { IImgCaption } from './IImgCaption';
+import styles from './imageCaption.module.css';
+
+const ImageCaption: FC<IImgCaption.IProps> = ({
+  imgCaptionLetter,
+  imgCaption,
+}): ReactElement => {
+  // make letter and devider (|) hidden when not pass letter as a props
+  const classes = classNames({
```

Comment:
Try to use more descriptive names for variables like `captionLetterClasses`.

Reply:
This name is based on Ayoub's comment. bcs it is one class for the style, there is no need for the long name

Reply:
Can't argue with that :)

Reply:
I think you mixing it up.
my comment was about the complexity of the name not being long. it would be nice to review my comment there (refer to our private chat). you used a negated naming while the condition contained another negation.

@AhmedAdel999 you shouldn't stop defending your argument just because my name is mentioned. if you think something isn't correct stand for it.

Reply:
![Screenshot_20210602-020253](https://user-images.githubusercontent.com/1198975/120403624-d691bd80-c344-11eb-8883-0aa661f7009e.jpg)